### ISSUE TITLE
Allow Multiple Directories Per Show

### ISSUE THREAD

**Issue by jchristisen** [pymedusa/Medusa#7319]
I have about 12tb worth of storage stashed across several drives some of them I use for live recordings from my Plex DVR it would be great if I could give it multiple directories to go look for a show. 

For instance I have we one place I store live recording from plex when I don't pull it down with Medusa. It would be great to be able to have more locations for a single show

**Comment by p0psicles** [comment]
How would you see this implemented?

**Comment by mysterfr** [comment]
Hi,

I'm joining this feature request.

Here's how I do stuff:
- seedbox downloads stuff (rtorrent)
- stuff is copied over from the seebox to my storage
- medusa is configured to download episodes torrent files into a specific directory
- I have an homemade queue manager (python script) that checks multiple folders (not only Medusa's output folder) and moves the torrents in the "watch" folder
- torrent is downloaded, then seeded for a long time
- after seeding target is met, torrent is removed, and the files are moved to a "repository"
I'm also sharing my seedbox with several users. 

Medusa is configured to look at the final repository folders for existing episodes and downloads what's missing.

It sometimes happens that one of the users puts an episode for download before Medusa does.
I would like for instance to be able to tell Medusa to not only look in my final repository, but also in rtorrent's "seed" directory.

So to answer @p0psicles' question, in the show configuration page, Medusa would for instance offer the option to add other lookup directory, maybe in a way as "preferred keywords" are entered : one at a time, with a "+" button.

Medusa would then have to look into the listed folders sequentially to check whether an episode already exists somewhere.

**Comment by p0psicles** [comment]
That should be done through the postprocessing process.

**Comment by mysterfr** [comment]
@p0psicles Sorry, but how could postprocessing take care of checking multiple folders before an episode gets snatched ?
Isn't post-processing meant to take place after the episode got downloaded ?

**Comment by p0psicles** [comment]
Medusa can periodically check a specific folder for now files to postprocess.
You have to make sure that all torrent downloaded files, get dropped into that folder. And medusa will try to process it and put it in the correct folder.

**Comment by mysterfr** [comment]
Thanks for your time @p0psicles !

I see, I guess I'll have to look again into post-processing...
I currently have it disabled because it doesn't match my use case.

Currently I work like this :
- torrent file are put in rtorrent's "watch" folder. torrent files can come from Medusa, Flexget or manual drop in a folder.
- my seedbox users can also directly put a torrent to download through ruTorrent web interface.
- torrent is downloaded and kept in seed for a varying time (combination of ratio/seedtime targets, varies from 3 weeks to 6 months) using pyrocore. During the seeding phase, all files are in a "seed" directory.
- when the torrent is removed files are moved to a "done" directory and stay there for 2 weeks before getting automatically deleted.

In parallel, I have my own dispatching Bash script, running every hour, that looks for TV episodes in both the "seed" and "done" folders, and dispatch them to various trees on my NAS, based on language (btw, see my Feature request https://github.com/pymedusa/Medusa/issues/8731).

I don't want/need Medusa to take care of moving files around, nor do I want it to rename anything.
I just need it be able to look into different locations before downloading something.

Do you still think post-processing can be applied in my use case ?
If so, would you mind elaborating your answer a bit as I'm a bit confused as to how I could implement what you suggested in your previous answer.

Thanks again !

**Comment by medariox** [comment]
Duplicate of https://github.com/pymedusa/Medusa/discussions/8731
